### ISSUE TITLE
Error on Find window rowconfigure

### ISSUE THREAD

**Issue by BartholomaeusAstrum** [SwatPhonLab/UltraTrace#131]
Functions just fine, and looks okay, but I get the error below whenever I open the find window (so far only with Command-F). The next line in the code also calls `Grid` which doesn't seem to be throwing an error? So not sure what's up with that. 

```
Exception in Tkinter callback
Traceback (most recent call last):
  File "/opt/anaconda3/envs/three-six/lib/python3.6/tkinter/__init__.py", line 1705, in __call__
    return self.func(*args)
  File "/Users/ZivStern/Desktop/UltraTrace/ultratrace/modules/textgrid.py", line 968, in openSearch
    self.app.Search.openSearch()
  File "/Users/ZivStern/Desktop/UltraTrace/ultratrace/modules/search.py", line 51, in openSearch
    self.createWindow()
  File "/Users/ZivStern/Desktop/UltraTrace/ultratrace/modules/search.py", line 46, in createWindow
    Grid.rowconfigure(self.window, 2, weight=1)
NameError: name 'Grid' is not defined
```

**Comment by mr-martian** [comment]
Looks like the search module just forgot to import `Grid` from `tkinter`. Should be fixed now.

**Comment by BartholomaeusAstrum** [comment]
Yup! Thank you! Closed by 4ba8dbe78e45b4f6ec1c038b3d4b1363477a92ba
